### PR TITLE
Update sslcrtd_program configuration directive documentation

### PR DESCRIPTION
**Pull request by marcosfrm**
Catch it up with 505f9427c55a365b464f1b204880df1d3ccccb7f.

### CODE REVIEW

File: src/cf.data.pre [hunk 1]
```diff
@@ -3268,7 +3268,12 @@ LOC: Ssl::TheConfig.ssl_crtd
 DOC_START
 	Specify the location and options of the executable for certificate
 	generator.
-	@DEFAULT_SSL_CRTD@ program requires -s and -M parameters
+
+	@DEFAULT_SSL_CRTD@ program can use a disk cache to improve response
+	times on repeated requests. In this mode, -s and -M parameters are
+	required. When neither parameters are given it operates without a
```
Comment (rousskov) [inline src/cf.data.pre]: I would polish the last two sentences a bit (shown here without formatting):

To enable caching, specify -s and -M parameters. If those parameters are not given, the program generates a new certificate on every request.

**Comment by marcosfrm** [comment]
Done.

File: src/cf.data.pre (hunk 1 above)
Comment (yadij) [reply src/cf.data.pre]: These commas are now making the text bad English. Verbing, the subject.

The previous text comma was replacing a subject definitive "the".

Initially the disk cache was the subject for the first 1.5 sentences and parameters the subject for the second 1.5 sentences. Now we have three different subjects in one paragraph with a change for each sentence. So commas are only appropriate for lists - of which we have none here.

Comment (rousskov) [reply src/cf.data.pre]: I see nothing wrong with the current phrasing as far as English is concerned. I do not know exactly what you mean by "Verbing, the subject", but many introductory phrases, including the ones used in the text in question, may, should, or must be followed by a comma (depending on the specifics of each case). You can find [many](https://owl.english.purdue.edu/owl/resource/607/03/) [explanations](https://www.quickanddirtytips.com/education/grammar/can-you-start-sentence-preposition) on this subject,

> So commas are only appropriate for lists

I am sorry, but that statement is very wrong. If you do not believe me, you can easily [find](https://www.google.com/search?q=comma+in+sentences) plenty of authoritative references on the subject.

**Pull request closed without merging**